frontend-react/src/hooks/UseOktaMemberships.ts

```diff
@@ -263,5 +295,60 @@ export const useOktaMemberships = (
         }
     }, [!!authState, authState?.isAuthenticated]); // eslint-disable-line react-hooks/exhaustive-deps
 
+    // TODO Just do fetch in here for now, use API config if possible!
+    useEffect(() => {
```

Comment:
If nothing else, I'll clean this up and make it its own hook like `useMembershipServices` so `useOktaMemberships` _only_ handles the dispatches to update membership state and not fetching the services as well.